accounting/report.go

```diff
@@ -43,5 +43,46 @@ type HarmonyEntry struct {
 	Credit bool
 }
 
+// getPriceFunc is a function that gets the price of msat value at a given time.
+type getPriceFunc func(amtMsat, ts int64) (float64, error)
```

Comment:
I wonder if this will cause a problem in the long run. Accounting may be problematic due to rounding errors (floats can't represent base 10 values exactly [1]).

[1] https://stackoverflow.com/questions/3730019/why-not-use-double-or-float-to-represent-currency

Reply:
Yeah, was also wondering about this. The change would need to happen in the `fiat` package, so will probably follow up on that in a separate PR. Can be achieved by just expressing fiat value in cents with an int. 

A rpc endpoint which returns `fiat_cents` is a bit strange, but probably cleaner in the long run.